bindings/matrix-sdk-ffi/src/encryption.rs

```diff
@@ -397,8 +397,15 @@ impl Encryption {
     pub fn verification_state_listener(
         self: Arc<Self>,
         listener: Box<dyn VerificationStateListener>,
+        emit_current_value: bool,
     ) -> Arc<TaskHandle> {
         let mut subscriber = self.inner.verification_state();
```

Comment:
(You could also have `verification_state` call `subscribe_reset()` in place of `subscribe()`, and not have any change here — that would make it easier to test this at the SDK layer too)

Reply:
I'll upload some changes doing this once I've tested them in the Android client.

Reply:
> Removed that commit, went back to what we had before.

Why, what went wrong.

> You could also have verification_state call subscribe_reset()

Ah, that's the function I was looking for, not `next_now()`, thanks for reminding us.